airbyte-webapp/src/views/Connector/ConnectorDocumentationLayout/ResourceNotAvailable/ResourceNotAvailable.module.scss

```diff
@@ -0,0 +1,10 @@
+@use "scss/variables";
+
+.requestContainer {
```

Comment:
We should add an `padding: variables.$spacing-xl` to this still, since at the moment the resize panel doesn't add any padding to it's content, thus this can overlap not very nicely:

![screenshot-20230213-133839](https://user-images.githubusercontent.com/877229/218460012-162d6be1-30b7-4c08-a0e0-1a9e456467d9.png)